templates/adherent_profile/activity/_instances.html.twig

```diff
@@ -27,3 +27,33 @@
             </div>
     </article>
 </div>
+
+{% set pcMembership = app.user.politicalCommitteeMembership %}
+{% if pcMembership and pcMembership.politicalCommittee.isActive %}
```

Comment:
I think you need to also update the `if` into `adherent_profile/activity.html.twig`

Reply:
why?